test/JDBC/expected/TestSpatialPoint-vu-verify.out

```diff
@@ -92,87 +305,989 @@ varbinary
 010100000017D9CEF753D34740D34D6210585936C0
 0101000000000000000000F03F0000000000000040
 010100000017D9CEF753D34740D34D6210585936C0
+01010000007593180456965EC017D9CEF753D34740
+~~END~~
+
+
+SELECT * FROM CoordsFromGeom;
+GO
+~~START~~
+float#!#float
+47.651#!#-22.349
+1.0#!#2.0
+47.651#!#-22.349
+<NULL>#!#<NULL>
+47.651#!#-22.349
+1.0#!#2.0
+47.651#!#-22.349
+47.651#!#-22.349
+1.0#!#2.0
+47.651#!#-22.349
+-122.349#!#47.651
+~~END~~
+
+
+SELECT * FROM equal_geom;
```

Comment:
Please make sure to add ORDER BY clause in each select statement to output of the query consistent.

Reply:
Done